sdk/servicebus/service-bus/samples-v1/typescript/src/receiveMessagesStreaming.ts

```diff
@@ -26,27 +32,46 @@ export async function main() {
 
   // To receive messages from sessions, use getSessionReceiver instead of getReceiver or look at
```

Comment:
get -> create

Reply:
Apparently that's been outdated for a bit...will fix.

Reply:
Fixed.